### ISSUE TITLE
Move stuff over to SFML?

### ISSUE THREAD

**Issue by thyrgle** [gavanw/vqisosmall#5]
The project both uses GLUT, POCO for networking, and SFML for audio. SFML is cross-platform and capable of audio, windows (+ OpenGL backend), and networking. Plus, it would mean a newcomer just has to learn SFML instead of the three libraries. Maybe move stuff over to just SFML?

**Comment by gavanw** [comment]
Yep this was my intention down the road. Note that you might hit a few complications with the use of native GL functions (which require an openGL lib) and SFML wrappers (which require (static IIRC) linkage to an OpenGL lib, but does not provide native OpenGL access!).
